### PR TITLE
Session expires

### PR DESCRIPTION
**Pull request by YesicaNavaB**
## Summary

Session expires

## Trello Card

https://trello.com/c/yeSGiWhs

### CODE REVIEW

File: test/utils.test.js [hunk 1]
```diff
@@ -4,13 +4,34 @@ const { factory } = require('factory-girl'),
   config = require('../config'),
   bcrypt = require('bcryptjs'),
   { saltNumber } = config.common.bcrypt,
-  salt = bcrypt.genSaltSync(Number(saltNumber));
+  salt = bcrypt.genSaltSync(Number(saltNumber)),
+  request = require('supertest'),
+  app = require('../app');
 
 exports.token =
```
Comment (jersonlopez) [inline test/utils.test.js]: remove unncessary variable

File: test/users.test.js [hunk 2]
```diff
@@ -137,39 +118,57 @@ describe('user list test', () => {
   it('should get the user list', done => {
     testCreate('yesica@wolox.co', 'shdfgs345', '1565721770').then(() => {
       request(app)
-        .get('/users?page=1&pageSize=5')
-        .set({ Accept: 'application/json', Authorization: token })
+        .post('/users/sessions')
```
Comment (jersonlopez) [inline test/users.test.js:121]: good solution! 🥇

File: test/sessions.test.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      email: 'sofia@wolox.co',
+      password: 'yuli35624',
+      administrator: true
+    }).then(() =>
+      request(app)
+        .post('/users/sessions')
+        .send({ email: 'sofia@wolox.co', password: 'yuli35624' })
+        .set('Accept', 'application/json')
+        .then(response => {
+          delay(3000).then(() => {
```
Comment (jersonlopez) [inline test/sessions.test.js]: i suggest you only 2000

File: test/sessions.test.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+          expect(result.statusCode).toBe(503);
+          expect(result.body.message).toBe('invalid Token ');
+          done();
+        })
+    );
+  });
+});
+
+describe('sessions expires', () => {
+  it('should not start navigation because the token expired', done => {
```
Comment (jersonlopez) [inline test/sessions.test.js]: change description please

**Review by jersonlopez — CHANGES_REQUESTED** [review]

File: test/utils.test.js (hunk 1 above)
Comment (YesicaNavaB) [reply test/utils.test.js]: ok!

File: test/sessions.test.js (hunk 3 above)
Comment (YesicaNavaB) [reply test/sessions.test.js]: ok!

File: test/sessions.test.js (hunk 4 above)
Comment (YesicaNavaB) [reply test/sessions.test.js]: ok!

File: test/fixture/albumPhotos.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  title: 'accusamus beatae ad facilis cum similique qui sunt',
+  url: 'https://via.placeholder.com/600/92c952',
+  thumbnailUrl: 'https://via.placeholder.com/150/92c952'
+};
+
+const albumsPhotos = [{ ...albumPhotos }, { ...albumPhotos, title: 'ejemplo' }];
+
+module.exports = {
+  single: albumPhotos,
+  all: albumsPhotos
```
Comment (jersonlopez) [inline test/fixture/albumPhotos.js]: if is never used, remove it please

File: test/__mocks__/request-promise.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 const logger = require('../../app/logger'),
-  { albumPhotos } = require('../utils.test');
+  dataAlbums = require('../fixture/albumPhotos');
 
 const album = {
```
Comment (jersonlopez) [inline test/__mocks__/request-promise.js]: it should also be in the fixtures

**Review by jersonlopez — CHANGES_REQUESTED** [review]

File: test/__mocks__/request-promise.js (hunk 6 above)
Comment (YesicaNavaB) [reply test/__mocks__/request-promise.js]: ok!

File: test/fixture/albumPhotos.js (hunk 5 above)
Comment (YesicaNavaB) [reply test/fixture/albumPhotos.js]: ok!

**Review by jersonlopez — APPROVED** [review]
good job yes!! finally finished your training 💯 💯 💯

**Pull request merged**